docs/style-guide/lists/index.md

```diff
@@ -0,0 +1,240 @@
+    link: https://entethalliance.org/
+  - title: Hyperledger
+    subtitle: Hyperledger is an open source collaborative effort created to advance cross-industry blockchain technologies. It is a global collaboration, hosted by The Linux Foundation, including leaders in finance, banking, Internet of Things, supply chains, manufacturing and Technology. The foundation has some projects in it that work with the Ethereum stack.
+    link: https://hyperledger.org/
+sampleListThree:
+  - title: EEA
+    link: https://entethalliance.org/
+  - title: Hyperledger
+    link: https://hyperledger.org/
+sampleListFour:
+  - title: EEA
+    meta: John Owens
+    link: https://entethalliance.org/
+  - title: Hyperledger
+    meta: John Owens
+    link: https://hyperledger.org/
+---
+
+# Component Usage
+
+## Standard Lists
+
+Standard lists can be called anywhere in markdown by using
+
+```
+- Lorem ipsum
+- Si dolor et amet
+- nunc habitasse portitor demi
+```
+
+Ensure that the list is always preceeded with one of the following, taking care to ensure that your markup is semantic:
+
+```
+Your heading text {.list-title}
+# Your heading text {.list-title}
+## Your heading text {.list-title}
+### Your heading text {.list-title}
+#### Your heading text {.list-title}
+##### Your heading text {.list-title}
+###### Your heading text  {.list-title}
+```
+
+{.mb-4}
+
+--- {.mb-4}
+
+### Examples{.mt-8}
+
+#### Unordered List {.list-title .mt-4}
+
+- Lorem ipsum si dolor et amet
+- Nunc habitasse et portitor demi
+
+Markdown:
+
+```
+#### List Example One {.list-title}
+- Lorem ipsum si dolor et amet
+- Nunc habitasse et portitor demi
+```
+
+#### Links with descriptions {.list-title .mt-8}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+  {.list-link-description}
+
+Markdown:
+
+```
+#### Links with descriptions {.list-title}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+{.list-link-description}
+```
+
+#### Links with descriptions without bullets {.list-title .mt-8}
+
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+- [Nunc habitasse et portitor demi](#) An example link followed by text
+  {.no-bullets .list-link-description}
```

Comment:
What's the difference between this (`{.no-bullets .list-link-description}`) & the update you made on /developers.index.md (`{.link-list .no-bullets}`)? I guess what I'm asking is what's the difference between `.link-list` & `.list-link-description`?

Reply:
Visually it looks the same (I think).

Reply:
Yeah there's no difference, good catch.

- `.link-list` made the `a` a block element to force a newline, which made the click target full width
- `.list-link-description used `.flex-column` to force a new-line, but that meant no inline elements could be used like `em`, `b` etc.

I'm moving this to only use `.link-list`, with a pseudo-element after the `a` to force a newline, instead of making it a block element.